docs/installation.md

```diff
@@ -52,17 +54,33 @@ Also make sure that your Podfile deployment target is set to >= 13.0 at the top
 
 ```ruby
 platform :ios, '13.0'
+...
+# if you get erros because of missing react-native-google-maps files e.g. RCTConvert+AirMap.h, consider adding the following 2 statements 
+pod 'GoogleMaps'
+pod 'Google-Maps-iOS-Utils'
```

Comment:
This shouldn't be added, as it is part of the podspec included in react-native-maps.

Reply:
In my case, I think it helped fix the issue. If you don't agree, you can decline it.